sklearn/cluster/dbscan_.py

```diff
@@ -103,7 +106,7 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     # neighborhood of point i. While True, its useless information)
     if metric == 'precomputed':
         D = pairwise_distances(X, metric=metric)
-        neighborhoods = [np.where(x <= eps)[0] for x in D]
+        neighborhoods = np.array([np.where(x <= eps)[0] for x in D])
```

Comment:
No. The radius queries are now all consistently including the points on the edges in sklearn. The point that I raised was the integer test `len(neighborhood) <= min_samples` to detect core samples.